docs/en/getting-started/quickstart.md

```
@@ -0,0 +1,492 @@
+        "source": "9e42824c-569a-5b89-808f-8b8ece56476e",
+        "target": "af397680-8867-5ec5-9ba9-dddd39dfd7a8",
+        "relation": "contains",
+        "weight": null,
+        "all_weights": {},
+        "relationship_type": null,
+        "edge_info": {
+          "source_node_id": "9e42824c-569a-5b89-808f-8b8ece56476e",
+          "target_node_id": "af397680-8867-5ec5-9ba9-dddd39dfd7a8",
+          "relationship_name": "contains",
+          "updated_at": "2025-12-04 08:54:17"
+        }
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)